Sources/BarcodeHeroCore/Helpers/BHImageHelper.swift

```diff
@@ -35,13 +35,8 @@ class BHImageHelper {
 
         context.setShouldAntialias(false)
         
-        if let fillColor = options.fillColor {
-            context.setFillColor(fillColor)
-        }
-        
-        if let strokeColor = options.strokeColor {
-            context.setStrokeColor(strokeColor)
-        }
+        context.setFillColor(options.fillColor)
```

Comment:
When using the `CGColor` initializer, this could require the `if let` blocks again. I believe it returns an optional `CGColor`.